net/src/p2p_config.rs

```diff
@@ -85,22 +85,41 @@ impl P2pConfig {
             .expect("file is not a proper JSON of a P2pConfig struct")
     }
 
-    pub fn default_mock() -> Self {
-        P2pConfig::from_str(P2pConfig::DEFAULT_MOCK_CONFIG)
+    pub fn default_ipc() -> Self {
+        P2pConfig::from_str(P2pConfig::DEFAULT_IPC_CONFIG)
             .expect("Invalid backend_config json on P2pConfig creation.")
     }
 
-    pub fn default_ipc() -> Self {
-        P2pConfig::from_str(P2pConfig::DEFAULT_IPC_CONFIG)
+    pub fn default_mock() -> Self {
+        Self::named_mock("default-mock")
+    }
+
+    pub fn named_mock(network_name: &str) -> Self {
+        P2pConfig::from_str(&Self::named_mock_config(network_name))
             .expect("Invalid backend_config json on P2pConfig creation.")
     }
+
+    pub fn named_mock_config(network_name: &str) -> String {
+        format!(
+            r#"{{
+    "backend_kind": "MOCK",
+    "backend_config": {{
+        "networkName": "{}"
+    }}
+}}"#,
+            network_name
+        )
+    }
 }
 
 // statics
 impl P2pConfig {
-    pub const DEFAULT_MOCK_CONFIG: &'static str = r#"{
-    "backend_kind": "MOCK",
-    "backend_config": ""
+    pub const DEFAULT_MOCK_CONFIG: &'static str = r#"
```

Comment:
might as well remove this, since its not used anymore.

Reply:
`DEFAULT_MOCK_CONFIG` is still used in one place, but doesn't show up in the diff